frontend/app/views/spree/shared/_homepage_meta_tags.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if current_store.meta_keywords.present? || current_store.seo_title.present? %>
```

Comment:
please add this logic to https://github.com/spree/spree/blob/e41ad3ab12a776414e5826e8a68c14c9e250560f/core/app/helpers/spree/base_helper.rb#L43